addons/html_editor/static/src/main/list/list_plugin.js

```diff
@@ -632,7 +632,7 @@ export class ListPlugin extends Plugin {
             return;
         }
         // Detect if cursor is at beginning of LI.
-        if (closestElement(startContainer, "LI") !== closestLIendContainer) {
+        if (!startContainer || closestElement(startContainer, "LI") !== closestLIendContainer) {
```

Comment:
What is the use case ? (test name ?)

Reply:
We should receive a range but startContainer is null. It look strange

Reply:
it's when there is no valid previous cursor position. so deletebackward should basically remove the list